build.gradle

```diff
@@ -116,16 +117,6 @@ task functional(type: Test) {
   outputs.upToDateWhen { false }
 }
 
```

Comment:
Does this imply the loss of PMD functionality?  I see now reports now or are they generated elsewhere?  or are we consciously removing it to bring into line with data-store and def-store etc?

Reply:
No, it doesn't lose the PMD functionality, it now inherits that and the rulesets from the following dependency

`id 'uk.gov.hmcts.java' version '0.12.0'`